src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -142,7 +155,11 @@ const renderJiraCloudAndEnterpriseServer = async (res: Response, req: Request):
 	const gheServerSubscriptions = difference(subscriptions, ghCloudSubscriptions);
 
 	// Connections for GHCloud
-	const { installations, successfulConnections, failedConnections } = await getConnectionsAndInstallations(ghCloudSubscriptions, req);
+	const {
+		installations,
+		successfulConnections: successfulCloudConnections,
```

Comment:
Renamed these so there is a clear distinction between successful/failed connections for cloud vs server.